inc/namespace.php

```diff
@@ -425,7 +425,12 @@ function ( $error ) {
 		'stats'     => $stats,
 	];
 
-	$trace['metadata'] = redact_metadata( $metadata );
+	$annotations = [
+		'memoryUsage' => round( memory_get_peak_usage() / 1048576, 2 ), // Convert bytes to mb.
+	];
+
+	$trace['metadata']    = redact_metadata( $metadata );
```

Comment:
This shouldn't be aligned, as alignment can cause unnecessary code churn in future changes.
```suggestion
	$trace['metadata'] = redact_metadata( $metadata );
```